src/main/java/edu/harvard/hul/ois/fits/tools/embarc/EmbARC.java

```diff
@@ -0,0 +1,351 @@
+package edu.harvard.hul.ois.fits.tools.embarc;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.StringReader;
+import java.io.StringWriter;
+
+import org.jdom.Attribute;
+import org.jdom.Document;
+import org.jdom.Element;
+import org.jdom.Namespace;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.portalmedia.embarc.cli.Main;
+import com.portalmedia.embarc.cli.XmlWriterDpx;
+import com.portalmedia.embarc.parser.dpx.DPXFileListHelper;
+import com.portalmedia.embarc.parser.dpx.DPXMetadata;
+import com.portalmedia.embarc.parser.FileFormat;
+import com.portalmedia.embarc.parser.FileFormatDetection;
+import com.portalmedia.embarc.parser.MetadataColumn;
+import com.portalmedia.embarc.parser.dpx.DPXColumn;
+import com.portalmedia.embarc.parser.dpx.DPXFileInformation;
+
+import edu.harvard.hul.ois.fits.Fits;
+import edu.harvard.hul.ois.fits.FitsMetadataValues;
+import edu.harvard.hul.ois.fits.exceptions.FitsToolException;
+import edu.harvard.hul.ois.fits.tools.ToolBase;
+import edu.harvard.hul.ois.fits.tools.ToolInfo;
+import edu.harvard.hul.ois.fits.tools.ToolOutput;
+
+public class EmbARC extends ToolBase {
+	private boolean enabled = true;
+	private Fits fits;
+	private final static Namespace fitsNS = Namespace.getNamespace (Fits.XML_NAMESPACE);
+
+	private File inputFile;
+	private FileFormat fileFormat;
```

Comment:
Storing these values at the object level makes this tool not thread safe, which is a problem because tools can be run concurrently from multiple threads.